docs/user/advanced.rst

```diff
@@ -237,7 +237,7 @@ or persistent::
 .. note:: If ``verify`` is set to a path to a directory, the directory must have been processed using
   the c_rehash utility supplied with OpenSSL.
 
-This list of trusted CAs can also be specified through the ``REQUESTS_CA_BUNDLE`` environment variable.
+This list of trusted CAs can also be specified through the ``REQUESTS_CA_BUNDLE`` or ``CURL_CA_BUNDLE`` environment variables.
```

Comment:
This should probably be clearer that `CURL_CA_BUNDLE` is the fallback to `REQUESTS_CA_BUNDLE`. Could we add a few extra words or a small sentence to that effect?

Reply:
@nateprewitt made an attempt. LMK if you have better idea.